src/test/java/org/gridsuite/sensitivityanalysis/server/service/FilterServiceTest.java

```diff
@@ -94,16 +94,16 @@ private String initMockWebServer() throws IOException {
             @Override
             public MockResponse dispatch(RecordedRequest request) {
                 String requestPath = Objects.requireNonNull(request.getPath());
-                if (requestPath.equals(String.format("/v1/filters/%s/export?networkUuid=%s&variantId=%s", LIST_UUID, NETWORK_UUID, VARIANT_ID))) {
+                if (requestPath.equals(String.format("/v1/filters/export?ids=%s&networkUuid=%s&variantId=%s", LIST_UUID, NETWORK_UUID, VARIANT_ID))) {
```

Comment:
You need to add another test with a list with multiple UUID instead of just one.